docs/en/administration/upgrade.md

```diff
@@ -1,18 +1,18 @@
 # Upgrade
 
-The upgrade methods of different JuiceFS client are different, which are described below.
+Different JuiceFS clients have different upgrade methods, which are described below.
 
 ## Mount point
 
-The JuiceFS client only has one binary file, so to upgrade the new version, you only need to replace the old one with the new one.
+The JuiceFS client only has one binary file. So to upgrade the new version, you only need to replace the old one with the new one.
 
-- **Use pre-compiled client**: You can refer to the installation method of the corresponding system in [this document](../getting-started/installation.md#install-the-pre-compiled-client), download the latest client, and overwrite the old one.
+- **Use pre-compiled client**: You can refer to [this document](../getting-started/installation.md#install-the-pre-compiled-client) for installation method of the corresponding system, downloading the latest client and overwriting the old one.
```

Comment:
ditch the whole sentense and replace with:

Refer to [Install the pre-compiled client
](../getting-started/installation.md#install-the-pre-compiled-client) for details.